packages/schemas/src/types/log/hook.ts

```diff
@@ -5,4 +5,5 @@ export enum Type {
   TriggerHook = 'TriggerHook',
 }
 
+// TODO: Implement all hook events
```

Comment:
@gao-sun do we still need this todo?

Reply:
no need then - as long as we have issues tracked